### PR TITLE
add footer component

### PR DESCRIPTION
**Pull request by OksanaKhorushko**

### CODE REVIEW

File: src/components/Footer/Footer.scss [hunk 1]
```diff
@@ -0,0 +1,80 @@
+@import '../../scss/variables';
+
+hr {
+  margin: 40px 0;
+  opacity: 0.7;
```
Comment (Ihor-Kleponosov) [inline src/components/Footer/Footer.scss]: Remove zero

File: src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -1,6 +1,92 @@
...
   return (
-    <footer>Footer</footer>
+    <footer className="footer">
+      <div className="footer__wrapper container">
+        <div className="footer__primary">
+          <div className="footer__col">
+            <Logo />
+            <div className="social__media">
+              <ul className="social__media-list">
+                <li className="social__media-item">
```
Comment (Ihor-Kleponosov) [inline src/components/Footer/Footer.jsx:13]: If list, then list-item

File: src/components/Footer/Footer.jsx [hunk 3]
```diff
@@ -1,6 +1,92 @@
...
+                  <a>
+                    <Image src={DiscordImg} alt="Tvit" />
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div className="footer__col">
+            <nav className="footer__primary-navigation">
+              <ul className="footer__primary-navigation-list">
```
Comment (Ihor-Kleponosov) [inline src/components/Footer/Footer.jsx]: The same about "list"

File: src/components/Footer/Footer.scss [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+.footer {
+
+  &__wrapper {
```
Comment (Ihor-Kleponosov) [inline src/components/Footer/Footer.scss]: Empty rule???

File: src/components/Footer/Footer.scss [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+    &-navigation {
+
+      &-list {
+        display: flex;
+      }
+
+      &-item {
+        padding: 0 6px;
+         margin-left: 6px;
```
Comment (Ihor-Kleponosov) [inline src/components/Footer/Footer.scss]: Spacing

File: src/components/Footer/Footer.scss [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+  &-item {
+    margin-right: 16px;
+    opacity: .7;
+  }
+}
+
+.sabscribe {
+  display: flex;
+  justify-content: right;
```
Comment (Ihor-Kleponosov) [inline src/components/Footer/Footer.scss]: Change "right" to flex-end?

File: src/components/Footer/Footer.scss [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+  margin-top: 24px;
+
+  & input {
+    border: 1px solid $color-main;
+    background: rgba(255, 255, 255, 0.1);
```
Comment (Ihor-Kleponosov) [inline src/components/Footer/Footer.scss]: Remove zero

**Review by Silwero — APPROVED** [review]

**Pull request merged**